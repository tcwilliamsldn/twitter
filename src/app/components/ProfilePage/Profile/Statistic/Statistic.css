h3 {
    color: #737373;
    font-size: 14px;
    margin: 0;
    padding: 0;
    font-weight: normal;
}

.value {
    color: #3c73ff;
    font-size: 22px;
}
