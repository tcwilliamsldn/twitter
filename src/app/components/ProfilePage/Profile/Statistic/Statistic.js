import React from 'react';
import styles from './Statistic.css';

const Statistic = ({ title, value }) => (
    <div>
        <h3>{title}</h3>
        <div className={styles.value}>{value}</div>
    </div>
);

export default Statistic;
