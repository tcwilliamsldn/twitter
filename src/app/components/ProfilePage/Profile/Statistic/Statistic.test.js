import Statistic from './Statistic';
import React from 'react';
import { shallow } from 'enzyme';

describe('Statistic', function() {
    it('renders a title and a value', function() {
        const component = shallow(<Statistic title="Stat" value={100} />);
        expect(component).toMatchSnapshot();
    });
});
