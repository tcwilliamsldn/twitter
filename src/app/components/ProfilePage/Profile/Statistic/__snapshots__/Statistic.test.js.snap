// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statistic renders a title and a value 1`] = `
<div>
  <h3
    className="title"
  >
    Stat
  </h3>
  <div
    className="value"
  >
    100
  </div>
</div>
`;
